### ISSUE TITLE
Scrape training data

### ISSUE THREAD

**Issue by mttwong** [Cake-Boys/champ-select-advisor#1]
Scrape draft phases and game outcomes from the riot games api.  We should preserve bans, picks, order, and game outcome. Additionally, we should be able to pull this information given a specific game.

**Comment by engky19** [comment]
https://hackernoon.com/league-of-legends-predicting-wins-in-champion-select-with-machine-learning-6496523a7ea7

https://github.com/meraki-analytics/cassiopeia/blob/master/README.md

https://cassiopeia.readthedocs.io/en/latest/

**Comment by mttwong** [comment]
With Cassiopeia looks like downloading this data should be trivial.  The discussion should then be, do we want to download the data into some kind of database or cloud storage, or should we just pull the data as needed during training?

**Comment by engky19** [comment]
I think that will come down to whether we need it readily available or cost of options. Ideally we won't spend any money or at least keep cost low.

**Comment by engky19** [comment]
Cloud database storage can be expensive. We may be able to dump a file in google drive or even on git

**Comment by mttwong** [comment]
:+1: For google drive (Virginia Tech gives us unlimited google drive)
:-1: For git (not what git is designed for).  I've tried this and it is cumbersome.

I don't mind spending some money.  We can also start with a database that we host.  For this particular issue, I think maybe we just download to sqlite or locally, and handle cloud storage later on. 
 See Cake-Boys/champ-select-advisor#2.

**Comment by engky19** [comment]
You have a server we could use right? We could utilize that

**Comment by mttwong** [comment]
No, I donated that to work.  Also I don't want to have to manage the upkeep of something like that.

**Comment by mttwong** [comment]
Let's move discussion about storage to the other issue #2.
